spec/lib/overrides_spec.rb

```diff
@@ -49,7 +49,9 @@
       "/app/cells/decidim/forms/step_navigation_cell.rb" => "8b9e56ff2ab04160bff91f408d38ba82",
       "/app/cells/decidim/forms/step_navigation/show.erb" => "228e2fa97ae251d0179ccc28de5326f8",
       # commands
-      "/app/commands/decidim/forms/answer_questionnaire.rb" => "526c95457a382f6d2295aee0d300693d"
+      "/app/commands/decidim/forms/answer_questionnaire.rb" => "526c95457a382f6d2295aee0d300693d",
```

Comment:
We use this file to track overrides we make to `decidim` files.

If the original file content changes, the test will not run, and we have to review the original file and the overriden one and manually merge them.